add keyboard scrolling for multiselect component

rishichawda opened the pull request.

rishichawda committed:
✨ add keyboard scrolling for multiselect

prasanna1211 reviewed (changes requested):
There seems to be a minor bug in multi select.
Select any three items. Deselect any 2. Dropdown toggles off. On deselecting 3rd item dropdown toggles on.

rishichawda committed:
🐛 fix dropdown issue on multiselect

The dropdown bug which was discovered on the multiselect component while deselecting all items but one would close the dropdown and bring it back again while deselecting / removing the last item. This only happened when deselecting through clicking on chip's close icon. This was happening due to the click event propagating to the chip's parent which also responds to the click event and toggles the menu. The issue is fixed now by stopping the event from bubbling up to the parent element. #135

rishichawda committed:
🐛 fix style on mozilla

fix overlapping dropdown options and text overflow not applied properly on items if the label is too long.
